### PR TITLE
Add form validation on email input

### PR DESCRIPTION
**Pull request by koachcode**
#Form Validation 
1. Validation is done on email input, to only submit when email input is in lowercase
2. Add an error message to prompt the user to enter a lower case value for email input
3. Add a condition to remove error when user tries changes the email input value to lowercase

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -358,3 +357,35 @@ Object.keys(navLinks).forEach((k) => {
...
+  if (emailTest !== email.value) {
+    errorMsg.innerText = 'Email should be lowercase.';
+    if (formTest === false) {
+      errorMsg.classList.toggle('hideErr');
+    }
+    formTest = true;
+    // To remove the errMsg when email input is clicked
+    email.addEventListener('click', () => {
+      if (formTest === true) {
+        errorMsg.classList.toggle('hideErr');
```
Comment (andersonlebon) [inline script.js:385]: ## [OPTINAL]

- [ ]  To improve the design I would suggest you add some styling error interaction on the input that contains the validation error. Just consider it as a challenge that can improve your UI.  @Ginohmk  [see the challenge here](https://andersonlebon.github.io/Portfolio-w2/). 
image
image

**Review by andersonlebon — APPROVED** [review]
Hi @Ginohmk 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is now professional :heavy_check_mark: 
- Nice design ✔️ 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: script.js (hunk 1 above)
Comment (koachcode) [reply script.js:385]: THanks

Comment (billionsjoel) [reply script.js:385]: I would also love to challenge you about your local storage, when someone deletes one field and refreshes the page, the deleted data should not come back. 😄

Comment (andersonlebon) [reply script.js:385]: image 
But kindly revise the requirements of the localstore  milestone

**Pull request merged**